extensions/sequelize/README.md

```diff
@@ -0,0 +1,86 @@
+# @loopback/sequelize
```

Comment:
Please add experimental badge @shubhamp-sf

Reply:
✅ Done,
Added the stability section, to stay consistent as other members in `/extensions` space.